### DISCUSSION TITLE
Session storage quota exception

### DISCUSSION THREAD

**Discussion by assoft** [livewire/livewire#1587] [4. General]
This record is created every time the page is refreshed... It's content too large and I just received a quota error...

Why?

![image](https://user-images.githubusercontent.com/835239/93094205-46262680-f6aa-11ea-89a3-fb4a815bd256.png)

Error:
```
DOMException: Failed to execute 'setItem' on 'Storage': Setting the value of 'g6zba6xp3j4' exceeded the quota.
```

**Comment by assoft** [comment]
>

Temporary Solution:

```javascript
<script>
    document.addEventListener("livewire:load", () => {
        sessionStorage.clear();
    });
</script>
```

**Reply by silicahd** [reply]
thank you I updated to 2.2.3 but it still did not fix the issue. This fixed it for now but still sucks because I cant store anything in the session.

**Comment by calebporzio** [comment]
Ok v2.2.3 only uses sessionStorage if it HAS to, and even then there is a system to clean up old entries if the storage has exceeded the maximum

**Reply by silicahd** [reply]
I have the same issue updated but 2.2.3 did not fix it. The above workaround did the trick but now I cant save stuff in session as it gets cleared when accessing my category page.  Maybe I am missing something.
